src/BaseModel.py

```diff
@@ -114,7 +114,7 @@ def call(self, t, x):
 class IAEffectLoader(object):
     generates_stats = False
 
-    def __init__(self, var, filenames, days, counties):
+    def __init__(self, var, filenames, days, counties, predict_for=None):

```

Comment:
ia effect loader needs to repeat the last iaeffects if we want to predict into the future